AEM/optimum-habana-perf.md

```diff
@@ -112,14 +115,14 @@ QUANT_CONFIG=./quantization_config/maxabs_measure_include_outputs.json python ..
 
 Here is an example to quantize the model based on previous measurements for LLama2 or 3 models:
 
-Users could export different values to below enivironment variables to change parameters for benchmarking
+Users could export different values to below environment variables to change parameters for benchmarking
```

Comment:
Different values can be exported to the environment variables listed below to change parameters for benchmarking.

Reply:
use iGPT to improve the sentences accordingly.